docker/Dockerfile-webservice

```diff
@@ -0,0 +1,4 @@
+FROM cowbird:base
+LABEL description.short="Cowbird Web Service"
+
+CMD gunicorn --paste ${COWBIRD_CONFIG_DIR}/cowbird.ini --preload
```

Comment:
More of a question, but since the only difference between docker images are the commands, wouldn't using the same image with different entrypoints make more sense? Or are they expected to evolve differently over the course of the project?

Reply:
This way, docker-compose that uses them doesn't even need to worry about which entrypoint to call (no need to provide an explicit entrypoint in the config). If changes do happen at some point for any reason, nothing changes on compose side also.